TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/HardwareElement.java

```diff
@@ -0,0 +1,79 @@
+package org.firstinspires.ftc.teamcode.subsystems;
+
+import java.util.HashMap;
+
+public class HardwareElement {
```

Comment:
What is the purpose of this class? I understand what it does, but not why you would need it. Also, add some Javadoc comments, I think he wants all public types to be documented.

Reply:
Nevermind, understood via the Discord! The JavaDoc comment still stands though.